DfE.FindInformationAcademiesTrusts/Extensions/CategoriesOfConcernExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using DfE.FindInformationAcademiesTrusts.Data;
+
+namespace DfE.FindInformationAcademiesTrusts.Extensions;
+
+public static class CategoriesOfConcernExtensions
+{
+    public static string ToDataSortValue(this CategoriesOfConcern rating)
+    {
+        return rating.ToDisplayString().ToLowerInvariant().Trim();
+    }
+
+    public static string ToDisplayString(this CategoriesOfConcern rating)
```

Comment:
These are string extensions not CategoriesOfConcernExtensions semantically we should change it, but we can do that in a future iteration. Resolve this when acknowledged